Overview.html

```diff
@@ -2656,12 +2656,22 @@ <h3 id="http-network-or-cache-fetch"><span class="secno">5.5 </span>HTTP-network
   <p>If <var>credentials flag</var> is set, run these substeps:
 
   <ol>
-   <li><p>Modify <var>httpRequest</var>'s
-   <a href="#concept-request-header-list" title="concept-request-header-list">header list</a>
-   per HTTP State Management Mechanism.
-   <a href="#refsCOOKIES">[COOKIES]</a>
-   <!-- XXX proper hook? -->
+   <li>
+    <p>If the user agent is not configured to block cookies for <var>httpRequest</var> (see
+    <a href="https://tools.ietf.org/html/rfc6265#section-7">section 7</a> of
+    <a href="#refsCOOKIES">[COOKIES]</a>), then run these substeps:
 
+    <ol>
+     <li><p>Let <var>cookies</var> be the result of the user agent's cookie store and
+     <var>httpRequest</var>'s <a href="#concept-request-current-url" title="concept-request-current-url">current url</a> into the
+     "cookie-string" algorithm (see <a href="https://tools.ietf.org/html/rfc6265#section-5.4">section 5.4</a>
+     of <a href="#refsCOOKIES">[COOKIES]</a>).
```

Comment:
This sentence reads a little weird I think. Perhaps 'be the result of running the "cookie-string" algorithm (...) with the user agent's cookie store and _httpRequest_'s current url".